<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AVR Avogadro</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="stepButton">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="registerTab">
       <attribute name="title">
        <string>Registers</string>
       </attribute>
       <layout class="QVBoxLayout" name="registerLayout">
        <item>
         <widget class="QGroupBox" name="registerGroupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Registers</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="pcLabel">
               <property name="text">
                <string>Program counter</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="NumericEdit" name="pcEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="instructionLabel">
               <property name="text">
                <string>Instruction</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="NumericEdit" name="instructionEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <layout class="QHBoxLayout" name="decodedInstHorizontalLayout">
               <item>
                <widget class="QLabel" name="decodedInstructionLabel"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="stackPointerLabel">
               <property name="text">
                <string>Stack pointer</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="NumericEdit" name="stackPointerEdit">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>StatusRegister (SREG)</string>
             </property>
             <layout class="QHBoxLayout" name="checkboxLayout">
              <item>
               <widget class="QCheckBox" name="iCheckBox">
                <property name="text">
                 <string>I</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="tCheckBox">
                <property name="text">
                 <string>T</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="hCheckBox">
                <property name="text">
                 <string>H</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="sCheckBox">
                <property name="text">
                 <string>S</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vCheckBox">
                <property name="text">
                 <string>V</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="nCheckBox">
                <property name="text">
                 <string>N</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="zCheckBox">
                <property name="text">
                 <string>Z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cCheckBox">
                <property name="text">
                 <string>C</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="RegisterWidget" name="registerWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="memoryTab">
       <attribute name="title">
        <string>Memory</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QHexEdit" name="hexEdit" native="true">
          <property name="toolTip">
           <string>QHexEditWidget</string>
          </property>
          <property name="whatsThis">
           <string>QHexEdit widget allow to edit the data in hex view.</string>
          </property>
          <property name="asciiArea" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="loadProgamFileMenuAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="gettingStartedMenuAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadProgamFileMenuAction">
   <property name="text">
    <string>&amp;Load memory</string>
   </property>
   <property name="iconText">
    <string>Load memory</string>
   </property>
  </action>
  <action name="actionGetting_started">
   <property name="text">
    <string>Getting started</string>
   </property>
  </action>
  <action name="gettingStartedMenuAction">
   <property name="text">
    <string>Getting started</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RegisterWidget</class>
   <extends>QWidget</extends>
   <header>RegisterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NumericEdit</class>
   <extends>QLineEdit</extends>
   <header>NumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header>qhexedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
